tsconfig.json

```diff
@@ -34,7 +34,10 @@
     "tests-ui/**/*",
     "global.d.ts",
     "eslint.config.ts",
-    "vite.config.mts",
-    ".storybook/**/*"
+    ".storybook/**/*",
+    "*.config.mts",
+    "*.config.ts",
+    "scripts/**/*.ts",
+    "build/**/*.ts"
```

Comment:
I think these directories should get their own tsconfigs instead of having them bundled with the root one.